WordPress/Classes/ViewRelated/Stats/Charts/Charts+Support.swift

```diff
@@ -18,6 +18,8 @@ extension BarChartDataSet {
 
 extension NSUIColor {
     static let chartColor = UIColor(red: 135/255.0, green: 166/255.0, blue: 188/255.0, alpha: 255.0/255.0)
+
+    static let highlightColor = UIColor(red: 245/255.0, green: 131/255.0, blue: 53/255.0, alpha: 255.0/255.0)
```

Comment:
Should we refer to named colors from `WPStyleGuide`?

Reply:
The colors were obtained from Zeplin; I was unable to find exact matches in the style guide. There were a couple of considerations that influenced this approach:

- The type (i.e., `NSUIColor`).
- The `chartColor` above is not part of the style guide, but is used [elsewhere](https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/WordPress/Classes/ViewRelated/Reader/WPStyleGuide%2BReader.swift#L55-L58) via a Reader-specific extension.

As for the highlight color here, I'll confer with @SylvesterWilmott about alternatives.

Reply:
I have updated the colors based on discussion via `p1551456028012400-slack-nde-stats-refresh`.